spec/lib/reporting/identity_verification_report_spec.rb

```diff
@@ -612,5 +612,17 @@
         subject.cloudwatch_client
       end
     end
+
+    describe '#verified_user_count' do
+      let!(:profile) do
+        create(
+          :profile, :active, created_at: time_range.end.end_of_day,
+                             verified_at: time_range.end.end_of_day
```

Comment:
personal style, I stick to "all same lines or all different lines". Not a requirement, just a suggestion
```suggestion
          :profile,
          :active,
          created_at: time_range.end.end_of_day,
          verified_at: time_range.end.end_of_day,
```